source/simulator/assemblers/stokes.cc

```diff
@@ -608,8 +608,6 @@ namespace aspect
           !=
           this->get_boundary_traction().end())
         {
-          scratch.face_finite_element_values.reinit (scratch.cell, scratch.face_number);
```

Comment:
Hm, why do we not need this any more? Because `reinit` is already called from the function that calls the current function?

Reply:
This was a bug, we used to call reinit twice (inside and outside of this function)!